src/utils/additional-headers.ts

```diff
@@ -16,5 +18,13 @@ export const getAdditionalHeaders = ({
     header['x-ttg-client-version'] = sourceVersion
   }
 
+  if (anonymousId) {
+    header['x-tt-anonymous-id'] = anonymousId
```

Comment:
```suggestion
    header['x-ttg-anonymous-id'] = anonymousId
```

Reply:
unfortunately no ... it's already defined as X-TT-Anonymous-ID in the platform, and all of our request headers use this pattern already, so i'd rather not change it. it's actually introducing inconsistency if we change it 😅